### PR TITLE
[script.kodi.android.update] 1.1.5

### PR DESCRIPTION
**Pull request by Lunatixz**
### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [script.foo.bar] 1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: script.kodi.android.update/addon.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="script.kodi.android.update" name="Kodi Android Installer" version="1.1.1" provider-name="Team Kodi">
+<addon id="script.kodi.android.update" name="Kodi Android Installer" version="1.1.5" provider-name="Team Kodi">
     <requires>
         <import addon="xbmc.python" version="2.24.0"/>
         <import addon="script.module.simplecache" version="1.0.0"/>
```
Comment (enen92) [inline script.kodi.android.update/addon.xml]: can you set the license to a spdx identifier?

File: script.kodi.android.update/select.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+LANGUAGE      = REAL_SETTINGS.getLocalizedString
+
+## GLOBALS ##
+DEBUG         = REAL_SETTINGS.getSetting('Enable_Debugging') == 'true'
+CUSTOM        = REAL_SETTINGS.getSetting('Custom_Manager')
+
+def log(msg, level=xbmc.LOGDEBUG):
+    if DEBUG == False and level != xbmc.LOGERROR: return
+    if level == xbmc.LOGERROR: msg += ' ,' + traceback.format_exc()
+    xbmc.log(ADDON_ID + '-' + ADDON_VERSION + '-' + (msg.encode("utf-8")), level)
```
Comment (enen92) [inline script.kodi.android.update/select.py]: have you tried this in Matrix? msg.encode("utf-8") will probably fail in py3

File: script.kodi.android.update/service.py [hunk 3]
```diff
@@ -38,37 +39,50 @@ def log(msg, level=xbmc.LOGDEBUG):
...
+        
+            
+    def getPlatform(self): 
+        log('getPlatform')
         count = 0
-        build = ''
         try:
-            while not xbmc.Monitor().abortRequested() and count < 3:
-                count += 1
+            while not xbmc.Monitor().abortRequested() and count < 15:
```
Comment (enen92) [inline script.kodi.android.update/service.py]: maybe create and reuse the same `xbmc.Monitor()` object

Comment (Lunatixz) [reply script.kodi.android.update/service.py]: I agree... lazy functions to get the job done. I'll include a single call in the next iteration.

File: script.kodi.android.update/select.py (hunk 2 above)
Comment (Lunatixz) [reply script.kodi.android.update/select.py]: Untested... I believe it might. Let me check it out later and i'll push changes.

**Comment by Lunatixz** [comment]
@enen92 should be good to go

**Pull request closed without merging**